service/http.proxy/src/main/java/org/kaazing/gateway/service/http/proxy/HttpProxyService.java

```diff
@@ -73,4 +86,37 @@ private void checkForTrailingSlashes(ServiceContext serviceContext) {
     protected HttpProxyServiceHandler createHandler() {
         return new HttpProxyServiceHandler();
     }
+
+    /**
+     * Method returning true if overriding accept/connect options found
+     * @param serviceContext
+     * @return
+     */
+    private boolean noOverridingAcceptConnectOptions(ServiceContext serviceContext) {
+        AcceptOptionsContext acceptOptionsContext = serviceContext.getAcceptOptionsContext();
+        ConnectOptionsContext connectOptionsContext = serviceContext.getConnectOptionsContext();
+
+        boolean bindsEmpty = acceptOptionsContext.getBinds().isEmpty();
+        boolean hasAcceptTransport = hasTransport(acceptOptionsContext.asOptionsMap());
+        boolean hasConnectTransport = hasTransport(connectOptionsContext.asOptionsMap());
+
+        return bindsEmpty && !hasAcceptTransport && !hasConnectTransport;
+    }
+
+    /**
+     * Method returning whether option list has *.transport(s)
+     * @param map
+     * @return
+     */
+    private boolean hasTransport(Map<String, Object> map) {
```

Comment:
Regarding *.bind option:
- for AcceptOptionsContext: I am checking the fact that the getBinds() method returns an empty list - so this is covered
- for ConnectOptionsContext: DefaultConnectOptionsContext seems to not specifically address any *.bind option (DefaultConnectOptionsContext asOptionsMap() method does not return any *.bind). I do see that however the gateway-config-201409.xsd file does allow tcp.bind for connect options. Is there a reason why the *.xsd seems to allow the tcp.bind for connect options, but this option seems to not further be used in the code?

In case the tcp.bind option is not used for connects, the *.bind scenario should be covered by the validation of:
>
boolean bindsEmpty = acceptOptionsContext.getBinds().isEmpty();